content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -197,6 +228,32 @@ Datadog uses the distributed tracing protocol and sets up the following HTTP hea
 
 `x-datadog-sampling-priority: 1`
 : To make sure that the Agent keeps the trace.
+{{% /tab %}}
+{{% tab "W3C Trace Context" %}}
+`traceparent: [version]-[trace id]-[parent id]-[trace flags]`
+: version: The current specification assumes version is set to 00.
+: trace id: 128 bits trace ID, expressed as padded hexadecimal
+: parent id: 64 bits span ID, expressed as padded hexadecimal
+: trace flags: Sampled (01) or not sampled (00)
+
+Example:
+: `traceparent: 00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331s-01`
+{{% /tab %}}
+{{% tab "b3 / b3 Multiple Headers" %}}
+`b3: [trace id]-[span id]-[sampled]`
+: trace id: 64 bits trace ID, expressed as padded hexadecimal
```

Comment:
Why add "padded" to the description?

Reply:
Datadog's IDs are decimal and not padded. Maybe you have a better formulation in mind?